files/en-us/web/api/request/arraybuffer/index.html

```diff
@@ -0,0 +1,63 @@
+---
+title: Request.arrayBuffer()
+slug: Web/API/Request/arrayBuffer
+tags:
+  - API
+  - ArrayBuffer
+  - Fetch
+  - Method
+  - Reference
+  - Request
+browser-compat: api.Request.arrayBuffer
+---
+<div>{{APIRef("Fetch")}}</div>
+
+<p>The <strong><code>arrayBuffer()</code></strong> method of the {{domxref("Request")}} interface
+  takes a {{domxref("Request")}} stream and reads it to completion. It returns a promise
+  that resolves with an {{jsxref("ArrayBuffer")}}.</p>
+
+<h2 id="Syntax">Syntax</h2>
+
+<pre class="brush: js">
+request.arrayBuffer().then(function(buffer) {
+   // do something with the buffer
+});
+</pre>
+
+<h3 id="Parameters">Parameters</h3>
+
+<p>None.</p>
+
+<h3 id="Return_value">Return value</h3>
+
+<p>A promise that resolves with an {{jsxref("ArrayBuffer")}}.</p>
+
+<h2 id="Examples">Examples</h2>
+
+<pre class="brush: js">
+const myArray = new ArrayBuffer(512);
```

Comment:
```suggestion
const myArray = new Uint8Array(10);
```

The indirection of `longInt8View` doesn't really make this example clearer IMHO. In fact by using a `Uint8Array` it's more clear that one can get an `ArrayBuffer` out even if there wasn't one to begin with.